pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -212,5 +212,12 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if errs := logs.ValidateLoggingConfiguration(&kc.Logging, field.NewPath("logging")); len(errs) > 0 {
 		allErrors = append(allErrors, errs.ToAggregate().Errors()...)
 	}
+
+	if localFeatureGate.Enabled(features.MemoryQoS) {
```

Comment:
The usual pattern here is:

if gate enabled && nil {
  // error: required value
}
if !nil {
    // check values
}

Reply:
Updated.